addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8640,8 +8655,10 @@ msgctxt "#19056"
 msgid "New timer"
 msgstr ""
 
+#. Label for a context menu entry to open timer settings dialog
+#: xbmc/pvr/windows/GUIWindowPVRTimers.cpp
 msgctxt "#19057"
-msgid "Edit timer"
+msgid "Settings"
```

Comment:
Well, as this string is only used in timer's context menus it imo is pretty clear that "Settings" here denotes the settings of the selected timer - this is in the nature of a context menu, imo. And actually this menu item opens the timer settings dialog which also says " Timer settings" in its heading.

If at other places a context menu item "Settings" brings you to system settings, then the text of that item should be changed to name the actual "out of context action" it triggers - "System Settings", right?